packages/amplify-graphql-searchable-transformer/src/cdk/create-searchable-domain.ts

```diff
@@ -1,13 +1,13 @@
 import { TransformerContextProvider } from '@aws-amplify/graphql-transformer-interfaces';
-import { EbsDeviceVolumeType } from '@aws-cdk/aws-ec2';
-import { CfnDomain, Domain, ElasticsearchVersion } from '@aws-cdk/aws-elasticsearch';
-import { IRole, Role, ServicePrincipal } from '@aws-cdk/aws-iam';
+import { EbsDeviceVolumeType } from 'aws-cdk-lib/aws-ec2';
+import { CfnDomain, Domain, ElasticsearchVersion } from 'aws-cdk-lib/aws-elasticsearch';
```

Comment:
The `Domain` from `aws-cdk-lib/aws-elasticsearch`. The API docs suggest to use `aws-cdk-lib/aws-opensearchservice`.
I took a stab at this but that resulted in non-trivial changes in CFN templates structure that was caught in unit tests. I.e.:
- resource type changed from `AWS::Elasticsearch::Domain` to `AWS::OpenSearchService::Domain`
- `RequestMappingTemplate` structure changed to different format.

I think that we should keep this as is, i.e. use deprecated API and address this post migration (but before CDK v3 where deprecated API is supposed to disappear).